docs/index.rst

```diff
@@ -4,4 +4,315 @@ JDBC Connector
+The database is monitored for new or deleted tables and adapts automatically. When copying data
+from a table, the connector can load only new or modified rows by specifying which columns should
+be used to detect new or modified data.
+
+Quickstart
+----------
+
+To see the basic functionality of the connector, we'll copy a single table from a local SQLite
+database. In this simple example, we'll assume each entry in the table is assigned a unique ID
+and is not modified after creation. Start by creating a database (you'll need to install SQLite
+if you haven't already)::
+
+   $ sqlite3 test.db
+   SQLite version 3.8.10.2 2015-05-20 18:17:19
+   Enter ".help" for usage hints.
+   sqlite>
+
+Next in the SQLite command prompt, create a table and seed it with some data::
+
+   sqlite> CREATE TABLE accounts(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255) NOT NULL);
+   sqlite> INSERT INTO accounts(name) VALUES('alice');
+   sqlite> INSERT INTO accounts(name) VALUES('bob');
+
+Now, create a configuration file that will load data from this database in
+``quickstart-sqlite.properties``::
+
+   name=test-sqlite-jdbc-autoincrement
+   connector.class=io.confluent.connect.jdbc.JdbcSourceConnector
+   tasks.max=1
+   connection.url=jdbc:sqlite:test.db
+   mode=increasing
+   increasing.column.name=id
+   topic.prefix=test-sqlite-jdbc-
+
+The first few settings are common settings you'll specify for all connectors. The ``connection.url``
+specifies the database to connect to, in this case a local SQLite database file. The ``mode``
+indicates how we want to query the data. In this case, we have an auto-incrementing unique
+ID, so we select ``increasing`` mode where each query uses the largest ID previously seen to limit
+new results to newly created rows. The ``increasing.column.name`` setting specifies the column in
+the table that contains the auto-incrementing ID. Finally, we can control the names of the topics
+that each table's output is sent to with the ``topic.prefix`` setting. Since we only have one
+table, the only output topic in this example will be ``test-sqlite-jdbc-accounts``.
+
+Now, run the connector in a standalone Kafka Connect worker in another terminal (this assumes
+Avro settings and that Kafka and the Schema Registry are running locally on the default ports)::
+
+   $ ./bin/connect-standalone ./config/connect-standalone.properties quickstart-sqlite.properties
+
+You should see the process start up and log some messages, and then it will begin executing
+queries and sending the results to Kafka. In order to check that it has copied the data that was
+present when we started Kafka Connect, start a console consumer, reading from the beginning of
+the topic::
+
+   $ ./bin/kafka-avro-console-consumer --new-consumer --bootstrap-server localhost:9092 --topic test-sqlite-jdbc-accounts --from-beginning
+   {"id":1,"name":{"string":"alice"}}
```

Comment:
Added explanation of the format -- definitely fair that not everyone will be familiar with the encoding . I've also adjusted the column for name to be nullable because SQLite has a kind of weird data model that causes the column to appear nullable even when marked as `NOT NULL`... This is probably good anyway because it's a good opportunity to discuss the Avro JSON encoding a bit, which I think often throws people off.